modules/table/src/integrationTest/java/org/apache/ignite/internal/table/ItColocationTest.java

```diff
@@ -341,6 +341,7 @@ public NetworkMessage clone() {
 
         intTable = new InternalTableImpl(
                 QualifiedNameHelper.fromNormalized(SqlCommon.DEFAULT_SCHEMA_NAME, "TEST"),
+                2, // zone id.
```

Comment:
It's related too, but as local variable close to the constructor.

Reply:
Below in tests are the similar cases, I didn't note them all, because the proposal is optional and opinionated.